tests/foreman/api/test_contentviewfilter.py

```diff
@@ -321,71 +318,6 @@ def test_positive_create_with_module_streams(
         assert content_view.id == cvf.content_view.id
         assert cvf.type == 'modulemd'
 
-    @pytest.mark.tier2
```

Comment:
why is this test removed ?

Reply:
I probably mixed the removal with #8737.  I can fix the conflict during my final submit before moving it out draft state.